### DISCUSSION TITLE
Gökçenin işleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#249] [Zeka Sorusu]
Gökçenin bugün yapması gereken A,B,C,D,E ve F ile göstereceğimiz 6 tane farklı işi var. Bu işlerin hepsini yapması ve birer kere yapması gerekiyor. Aynı anda iki işi yapamıyor. İşlerin bazıları diğerlerine bağlı olduğu için aşağıdaki sıra koşullarına uyması gerekiyor.
● C’yi F’den sonra yapması gerekiyor.
● B’yi en sona bırakamıyor.
● D’yi ya B’den hemen önce ya da hemen sonra yapması gerekiyor.
● A’yı E’den hemen önce yapması gerekiyor.
Soru1: A’yı 4. sırada yaptığı biliniyorsa, B’yi kaç farklı zamanda (sırada) yapabilir?
Soru2: B’yi 5. sırada yaptığı ve E’yi de D’den hemen önce yaptığı biliniyorsa C’yi kaçıncı sırada yapmıştır?

**Comment by Adaisland** [comment]
Soru 1
1. _B_
2.F
3.C
4.**A**
5.E
6.
Sadece 1 sırada çünkü F ve C diğer türlü yerleşemez
Soru 2
1._F_
2.**C**
3.**E**  değiştirilemez
4.**D** değiştirilemez
5.**B**
6.
C ikinci sırada olabilir
